docs/node/testnet/upgrade-log.md

```diff
@@ -57,7 +57,13 @@ upgrade epoch.
 5. Ensure your node will remain stopped by disabling auto-starting via your
    process manager (e.g., [systemd] or [Supervisor])
 
-6. [Wipe state]. This must be performed _before_ replacing the Oasis Node
+6. Back up the entire directory of your node. Verify that the backup includes
+the following folders:
+- for consensus: `tendermint/abci-state` and `tendermint/data`
+- for runtimes: `runtimes/*/mkvs_storage.badger.db` and
+`worker-local-storage.badger.db`
```

Comment:
This would be less confusing IMO.
```suggestion
- for runtimes: `runtimes/*/mkvs_storage.badger.db` and
`runtimes/*/worker-local-storage.badger.db`
```
Or something like this (but people may find this strange):
```suggestion
- for runtimes: `runtimes/*/{mkvs_storage.badger.db,worker-local-storage.badger.db}`.
```
The easiest way would actually be to just backup the entire `runtimes` directory and that's it. The `history.db` is usually not that large compared to others.

Reply:
Added the first suggestion. :)
The first sentence says to backup the entire node directory. This list is just for verification that important files are in the backup.